### PR TITLE
Add Ryougi Shiki variants

### PR DESCRIPTION
**Pull request by deusexcalamus**
Add a couple variations for Ryougi Shiki in FGO/Type-Moon.

### CODE REVIEW

File: sopel_waifu/waifu.json [hunk 1]
```diff
@@ -2444,6 +2444,8 @@
         "Rani VIII",
         "Ryougi Mana",
         "Ryougi Shiki (F/GO)",
+        "Ryougi Shiki (Assassin)",
+        "Ryougi Shiki (Saber)",
```
Comment (dgw) [inline sopel_waifu/waifu.json:2448]:
```suggestion
        "Ryougi Shiki (Assassin) (F/GO)",
        "Ryougi Shiki (Saber) (F/GO)",
```

Without this, these new items won't appear in the fgowaifu command you like so much.

**Review by dgw — COMMENTED** [review]
The suggestion, I can apply myself when merging.

If these two new items don't replace the vanilla Ryougi Shiki (with nothing in parentheses), it's good to go.

**Comment by deusexcalamus** [comment]
oops lmao, the FGO entities are distinct from the normal one, you can go ahead and do how you suggested.

**Review by dgw — APPROVED** [review]
Ready when suggestion is applied. Will merge when I'm back at a PC.

**Pull request merged**